features/account/receiving_email_after_registration.feature

```diff
@@ -9,11 +9,29 @@ Feature: Receiving a welcoming email after registration
         And that channel allows to shop using "English (United States)" and "Polish (Poland)" locales
 
     @ui @email @api
-    Scenario: Receiving a welcoming email after registration
+    Scenario: Receiving a welcoming email after registration when channel has disabled registration verification
+        Given on this channel account verification is not required
         When I register with email "ghastly@bespoke.com" and password "suitsarelife"
-        Then a welcoming email should have been sent to "ghastly@bespoke.com"
+        Then only one email should have been sent to "ghastly@bespoke.com"
+        And a welcoming email should have been sent to "ghastly@bespoke.com"
+
+    @ui @email
```

Comment:
Why this scenario is not cover by API?
we should add `@api` tag or `@no-api` tag (same in line 32)